packages/calypso-e2e/src/lib/pages/gutenberg-editor-page.ts

```diff
@@ -327,4 +332,20 @@ export class GutenbergEditorPage {
 		await Promise.all( [ this.page.waitForNavigation(), frame.click( selectors.viewButton ) ] );
 		await this.page.waitForLoadState( 'networkidle' );
 	}
+
+	/**
+	 * Opens the Block Editor sidebar (left hand side).
+	 */
+	async toggleSidebar(): Promise< void > {
```

Comment:
Do we want to make this match the behavior of the settings sidebar where instead of just toggling, we have methods for opening and closing that only take action if the sidebar isn't already opened or closed (respectively?)

Toggling to me always feels a little risky, as you may not know the starting state!

Reply:
Will change to just `openSidebar` for now as no tests need to close it yet - we'll make it when that time comes.